package.json

```
@@ -9,7 +9,7 @@
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "test": "npm run test:unit && npm run test:typescript",
-    "test:unit": "tap",
+    "test:unit": "c8 -100 node --test",
```

need the double `-` for c8 [ref](https://github.com/bcoe/c8?tab=readme-ov-file#checking-coverage)
```suggestion
    "test:unit": "c8 --100 node --test",
```